### PR TITLE
Add changes from the ekstraordinary general assembly 2021

### PR DESCRIPTION
**Pull request by LudvigHz**

### CODE REVIEW

**Comment by ingridmaries** [comment]
Wow! Amazing

**Review by ingridmaries — APPROVED** [review]
Great changes, looks good!

**Review by erlingfn — DISMISSED** [review]
Looks good!

File: abakus-statutter/09-organisasjon.tex [hunk 1]
```diff
@@ -23,10 +23,10 @@ \subsubsection{}
 \item Sosialt: Arrkom, Koskom, LaBamba
 \item Inter: Fagkom, Bedkom
 \item Media: PR, readme, Webkom
-\item Diverse: backup
+\item backup, Bankkom
 \end{itemize}
 
-\subsection{}
+\sub
```
[the remark this answers is not in the document]
Comment (LudvigHz) [reply abakus-statutter/09-organisasjon.tex:29]: oisann. Her har det nok skjedd en feil :sweat_smile:.

_It was a test :tm:_

**Review by erlingfn — APPROVED** [review]
Looks good to me!

**Pull request merged**